platform/device/udidauth.go

```diff
@@ -102,5 +102,4 @@ func (mw *udidCertAuthMiddleware) Checkin(ctx context.Context, req mdm.CheckinEv
 	default:
 		return errors.Errorf("unknown checkin message type %s", req.Command.MessageType)
 	}
-	return mw.next.Checkin(ctx, req)
```

Comment:
How else is the next middleware layer going to be called?

Reply:
it's returned in the case statements. this is dead code anyway because of the `default`. 

since each switch case has a return, the last line was dead code.